backend/app/rest/api/rest_private.go

```diff
@@ -43,6 +44,9 @@ type privStore interface {
 	Vote(req service.VoteReq) (comment store.Comment, err error)
 	Get(locator store.Locator, commentID string, user store.User) (store.Comment, error)
 	User(siteID, userID string, limit, skip int, user store.User) ([]store.Comment, error)
+	GetStringUserDetail(locator store.Locator, userID string, detail engine.UserDetail) (string, error)
```

Comment:
I don't like this semi-generalized method, not the set one. As far as I understan they both hits storage service and service may implement very concrete getter/setters like GetUserEmail, SetUserEmail